.github/workflows/check.yml

```diff
@@ -25,3 +25,10 @@ jobs:
         env:
           MAPBOX_ACCESS_TOKEN: ${{ secrets.MAPBOX_ACCESS_TOKEN }}
           ABLY_API_KEY: ${{ secrets.ABLY_API_KEY }}
+
+      - name: Xcodebuild Logs Artifact
+        if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: xcodebuild-logs
+          path: ~/Library/Developer/Xcode/DerivedData/*/Logs
```

Comment:
If I understand your [comment](https://github.com/ably/ably-asset-tracking-swift/pull/193#issuecomment-951634212) correctly, this code was added only for some investigation reasons, right? Shouldn't we remove it then?